README.md

```diff
@@ -234,11 +234,11 @@ To disable leader election, define an env variable named ENABLE_LEADER_ELECTION
 
 **_Parameters:_**
 
-| Name            | Description                                                                                                                                                                  |                             Default                              |
-| --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :--------------------------------------------------------------: |
-| onDelete        | If it exists and has a delete value, delete the directory, if it exists and has a retain value, save the directory.                                                          | will be archived with name on the share: `archived-<volume.Name>` |
-| archiveOnDelete | If it exists and has a false value, delete the directory. if `onDelete` exists, `archiveOnDelete` will be ignored.                                                           | will be archived with name on the share: `archived-<volume.Name>` |
-| pathPattern     | Specifies a template for creating a directory path via PVC metadata's such as labels, annotations, name or namespace. To specify metadata use `${.PVC.<metadata>}`. Example: If folder should be named like `<pvc-namespace>-<pvc-name>`, use `${.PVC.namespace}-${.PVC.name}` as pathPattern. |                               n/a                                |
+| Name            | Description                                                  |                           Default                            |
+| --------------- | ------------------------------------------------------------ | :----------------------------------------------------------: |
+| onDelete        | If it exists and has a delete value, delete the directory, if it exists and has a retain value, save the directory. | will be archived with name on the share: `archived-<volume.Name>` |
+| archiveOnDelete | If it exists and has a false value, delete the directory. if `onDelete` exists, `archiveOnDelete` will be ignored. | will be archived with name on the share: `archived-<volume.Name>` |
+| pathPattern     | Specifies a template for creating a directory path via PVC metadata's such as labels, annotations, name or namespace. To specify metadata use `${.PVC.<metadata>}`. Example: If folder should be named like `<pvc-namespace>-<pvc-name>`, use `${.PVC.namespace}-${.PVC.name}` as pathPattern. The default pattern will add the pvname to folder, which will result in a unique folder name, combining namespace, name and an unique id. | `${.PVC.namespace}-${.PVC.name}-${.PVC.pvname}` |
```

Comment:
i am wondering maybe its good to mention the default in the [chart's README](https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner/blob/c2a2d5d544781e3d3e4d7a7e2d21a8e64cf6d9d1/charts/nfs-subdir-external-provisioner/README.md?plain=1#L66) as well. i mean, i would keep the `storageClass.pathPattern` null but mentions that null means default which is implicitly `${.PVC.namespace}-${.PVC.name}-${.PVC.pvname}`

Reply:
Hey, are we sure this is correct `${.PVC.pvname}`? It seems I'm not able to use that variable in the pathPattern, it's always empty

Reply:
done